app/lua_components/menu.lua

```diff
@@ -797,6 +797,12 @@ function CreateMenu(info)
 
             return t.Items[#t.Items] or item
         end,
+        --- Create child menu from properties of this object
+        ---@param t Menu|string MenuV menu
+        ---@param overrides table<string, string|number> Properties to override in menu object (ignore parent)
+        CreateChildMenu = function(t, overrides)
```

Comment:
`CreateChildMenu` should be `CloneMenu` or `InheritMenu` because child menu's don't exist and each menu is equal to each other.

Reply:
Ok, I'll change it to `InheritMenu`, since `CloneMenu` makes it sound like you're cloning menu items as well.

Reply:
`InheritMenu` is nice 👍